assets/eip-5851/contracts/interfaces/IERC5851.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: CC0-1.0
+pragma solidity ^0.8.0;
+
+interface IERC5851 {
```

Comment:
Where is this interface referred to from the standard?

Reply:
yes, this is the original version of the implementation. we were re-writing the details on the md file so you might not find them in the **Specification** section. i will push the final version soon